Themes/default/css/responsive.css

```diff
@@ -232,25 +245,58 @@
 	/* Display (Topics) */
 	.poster {
 		float: none;
+		width: auto;
+		position: relative;
 	}
 	.postarea {
 		margin: 0;
 	}
+	.inner {
+		padding: 1em 2px;
+	}
 	.moderatorbar {
 		margin: 0;
 	}
-	.keyinfo h5 .modified {
-		float: none;
+	.keyinfo {
+		padding-top: 5px;
+		margin-top: 5px;
+		clear: both;
+	}
+	.keyinfo .postinfo {
+		font-weight: normal;
+	}
+	.keyinfo .postinfo .smalltext,
+	.keyinfo .page_number {
+		opacity: 0.6;
+	}
+	.keyinfo .postinfo a.smalltext:hover {
+		opacity: 1;
 	}
 	img.icon, #forumposts .catbg img {
 		display: none;
 	}
-	.user_info li {
+	.poster h4 {
```

Comment:
I like this idea.
Do you think we should remove custom fields on mobile for this area?
This is a silly example:
![Screenshot_32](https://user-images.githubusercontent.com/3318127/150698762-a16f602e-1986-4e90-bb4b-0e1d293261ef.png)

Reply:
Perhaps, or make make them inline-block as well? I'm open to suggestions either way.

Reply:
It's not visible below avatar, with icons, and neither below the poster info. Perhaps we should hide it here too so it's consistent.

Reply:
Agreed

Reply:
Done in 8ecac6f